cmd/server/command/start.go

```diff
@@ -131,11 +133,29 @@ func NewStart(startOptions *startOptions) *cobra.Command {
 					},
 				},
 			}
+			// Import GPG Keys
+			if startOptions.gitConfigOpts.SigningKey != "" {
+				if len(startOptions.gitConfigOpts.GPGImportPaths) < 1 {
+					return errors.New("gpg signing key provided, but no import paths specified")
+				}
+				for _, p := range startOptions.gitConfigOpts.GPGImportPaths {
+					// lets just use flux' implementation on how to load keys
+					keyfiles, err := gpg.ImportKeys(p, false)
+					if err != nil {
+						return errors.New(fmt.Sprintf("failed to import GPG key(s) from %s", p))
```

Comment:
We can use `fmt.Errorf` for formatted errors.
```suggestion
						return fmt.Errorf("failed to import GPG key(s) from %s", p)
```